actionview/lib/action_view/lookup_context.rb

```diff
@@ -217,6 +217,13 @@ def normalize_name(name, prefixes)
 
         if !prefixes || prefixes.empty?
           prefixes = [path_prefix]
+
+          while (idx = path_prefix.index("/"))
+            path_prefix = path_prefix.from(idx)
+            path_prefix = path_prefix.from(1) if path_prefix.start_with?("/")
+
+            prefixes.push(path_prefix)
+          end
```

Comment:
@rafaelfranca the surrounding code was optimized by https://github.com/rails/rails/commit/5955a0a3f3eb67ef29cdec078f1d4bb53e9cd749 to move away from `String#split` and toward index-based access, so I followed that pattern for this loop.